jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -40,8 +40,11 @@
      (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
    (let ((use-torso t))
      (if (and (member :use-torso args) (null (cadr (member :use-torso args)))) (setq use-torso nil))
-     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t)
+     (send* self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t args)
    ))
+  (:rarm-angle-vector 
+   (av &optional (tm 3000) &rest args)
+   (send* self :angle-vector av tm :ctype :arm-controller args))
```

Comment:
*Very good discussion*

if you want to move only the arm,
```
(send *ri* :angle-vector av 3000 :arm-controller)
```
is correct syntax, and if it not working on the fetch, we should change something like
```
   (:angle-vector ;; this verison uses :angle-vector-sequence for sendding trajectory, this enable us to use :wait-interpolation method so we choose this for now
    (av &optional (tm 3000) (ctype controller-type) &rest args)
....
     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t :controller-type ctype)
```

Please check if this works.


> Doesn't seem to have a great demand so maybe doesn't really pay off defining a new method for it, but also didn't see any down-points for letting it defined.

The downside is defining new method for one particular robot, if someone learn to move fetch robot with `(send *ri* :rarm-angle-vector)` one expect other robots including pr2, baxter, hrp2, jaxon, kengoro, turtlerobot... will move in same way,

So if you want to propose new methods...

1. propose in pr2eus/robot-interface.l, all robots uses this file
2. since everyone knows `(send *robot* :rarm  :angle-vector)`,  `(send *ri* :rarm :angle-vector #f(...))` may get support from others.
3. If you propose new methods, it will take months to finally get merged. Because we have not accept that API for more than 20 years... So better to move on next topic